nri-kafka/src/Kafka/Test.hs

```diff
@@ -0,0 +1,251 @@
+-- | Tests, exposed so we can run them against GHCID
+module Kafka.Test
+  ( stub,
```

Comment:
💭 I think we should only expose `stub` from this public module. All the other helpers we wrote to support writing tests for this library itself. They're pretty slow and flaky and mess up test runner output. I think we might want to put them in a private module.

Reply:
Going to do it.